packages/esm-patient-common-lib/src/dashboards/dashboardextension.scss

```diff
@@ -0,0 +1,20 @@
+@import "~@openmrs/esm-styleguide/src/vars";
+@import "~carbon-components/src/globals/scss/vars";
+@import "~carbon-components/src/globals/scss/mixins";
+
+.activeNavItem {
+  background-color: $ui-03;
+  outline: none;
+  color: $ui-05;
+  border-left: 4px solid var(--brand-01);
+
+  a {
+    color: $ui-05 !important;
```

Comment:
Please don't use important

Reply:
Removed the `important` and removed the overriding styles from the [left-nav-styles in core](https://github.com/openmrs/openmrs-esm-core/pull/457).